### ISSUE TITLE
View button doesnt seem to be working - Xbox360 game - Fallout 3

### ISSUE THREAD

**Issue by flipside101** [redphx/better-xcloud-android#84]
### Device

Other

### Device name

ShieldTV

### Android Version

11 i think - SHIELD Experience 9.1

### App version

0.12.0

### Script version

4.0.0

### Reproduction Steps

open Fallout 3
Press the View button
Its should bring up the wait menu, nothing happens. It works on windows in edge using the script
both connected controller via Bluetooth

Just tested and it works on my Pixel 6 using the same app and script versions.

**Comment by redphx** [comment]
#81

**Comment by flipside101** [comment]
Just checking as reading back  I don't think I have explained this correctly , the View button I'm talking about Is one of the keys on the Xbox controller  , is this a  known issue with the AndroidTV setup

**Comment by KingNothing81** [comment]
@flipside101: I’m happy that I was able to help. I also got this information from a user here. =)

Yes, the “View“ button is like Microsoft calls the left side button on Xbox Controllers. But connected over Bluetooth to an Android device, it acts like the “Select” button. Microsoft also calls the right side button the “Menu” button. But on Android it acts like the “Start” button. It’s really confusing sometimes.

But Nvidia’s modification makes the left side button on controllers react like Android’s “Back” button and the middle button like “Home”. It seems that “real” apps can override this, but it causes problems when using xCloud in a web browser or WebView (like with the BXC app).

I suspect that the button mapping problems on some Fire TV devices have a similar reason. That Amazon has done a similar modification on those devices. But I don’t know how this could be disabled on Fire OS.
